.github/workflows/tag.yml

```diff
@@ -61,3 +61,26 @@ jobs:
             golangci/golangci-lint:${{ steps.prepare.outputs.tag_name }}-alpine
             golangci/golangci-lint:${{ steps.prepare.outputs.major_tag }}-alpine
             golangci/golangci-lint:latest-alpine
+  update_github_action:
+    needs: [release]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+         go-version: 1.15
+    - name: Update GitHub action config
```

Comment:
>  I test this on a dummy tag I created on my fork - https://github.com/neha-viswanathan/golangci-lint/tree/v100
https://github.com/neha-viswanathan/golangci-lint/actions/runs/315233729

Thanks, the above github action was kicked off successfully, but I don't think that we can confirm things will run as per expectation due to 2 reasons:
- update_github_action was skipped due to _release_ failure. For testing, you might need to relax this dependency.
- check-in-files step didn't do anything due to no changes in previous step.

Another point is that GITHUB_TOKEN environment might be required for these two steps.